src/section-factoring-by-grouping.ptx

```diff
@@ -71,10 +71,10 @@
             <p>Suppose we must factor <m>x^3-3x^2+5x-15</m>. Note that there are four terms, and they are written in descending order of the powers of <m>x</m>. <q>Grouping</q> means to group the first two terms and the last two terms together:<md>
                 <mrow>x^3-3x^2+5x-15\amp=\left(x^3-3x^2\right)+(5x-15)</mrow>
                 <intertext>Now, each of these two groups has its own greatest common factor we can factor out:</intertext>
-                <mrow>\amp=x^2(x-3)+5(x-3)</mrow>
-                <intertext>In a sense, we are <q>lucky,</q> because we now see matching binomials that can themseves be factored out:</intertext>
-                <mrow>\amp=x^2\attention{\overbrace{(x-3)}}+5\attention{\overbrace{(x-3)}}</mrow>
-                <mrow>\amp=(x-3)\left(x^2+5\right)</mrow>
+                <mrow>\phantom{x^3-3x^2+5x-15}\amp=\highlight{x^2}(x-3)\highlight{{}+5}(x-3)</mrow>
```

Comment:
I don't think all these phantoms you added are doing what you think they are doing. The goal was to align equals signs. But all this content is centered, not left-aligned. So if it is important to use phantoms to get that alignment, you have to identify the longest content appearing in a left side (what you have done) and also identify the longest content appearing on the right side (which seems missing.) And it may not always be possible to add just the right amount to a right side somewhere to get another block to align.